doc/developer-guides/doc_guidelines.rst

```diff
@@ -294,11 +294,21 @@ For example, there are roles for marking :file:`filenames`
 (``:command:`make```).  You can also use the \`\`inline code\`\`
 markup (double backticks) to indicate a ``filename``.
 
-Don't use items within a single backtick, for example ```word```.
+Don't use items within a single backtick, for example ```word```. Instead
+use double backticks: ````word````.
 
-For references to files that are in the ACRN Hypervisor GitHub tree, you can
-use a special role that creates a hyperlink to that file. For example, a
-GitHub link to the reST file used to create this document can be generated
+Branch-specific File Links
```

Comment:
Capital 'S' here (Branch-Specific), because it's a two-word hyphenate rather than a prefix or suffix situation like "Pre-release".